### ISSUE TITLE
Buffer list in "most recently visited" order

### ISSUE THREAD

**Issue by x-ji** [tridactyl/tridactyl#1040]
The buffer list in Vimium triggered by `T` lists the tabs in a most recently visited order. This can be more helpful than the default behavior of Tridactyl where all tabs are simply listed according to their order in the browser: Sometimes I easily have dozens of tabs open and I'd want to switch between several recent tabs quickly. If I want to go to a specific tab I'd directly search for it instead of going through the huge default list, whose ordering doesn't really serve any purpose in this case.

Even when I type in some search term (e.g. github) there can still be 10+ matching tabs. If the matches are also listed in the "most recently visited" order, I can get to the tab I want much more easily.

Is it possible to offer this as a configuration option? Since we can already jump to the last used tab it seems there should already be a stack of tabs stored somewhere.

**Comment by bovine3dom** [comment]
Sure, seems like a good idea. Blocked by #932.

**Comment by berkeleyinc** [comment]
I'd love to see this feature!
It's the only reason why I'm using another extension over tridactyl.

It doesn't seem to be blocked by #932 anymore.
@bovine3dom I haven't seriously looked into tridactyls source yet, but in case this Feature Request here is not a priority at all, I could at least hack some kind of alternative sort function if you could point me towards the right source locations

**Comment by bovine3dom** [comment]
@berkeleyinc sorry for the slow reply, I missed this somehow!

You essentially need to just not run this line based on a config switch: https://github.com/tridactyl/tridactyl/blob/9a0937a4f04f2c4aae1033a66ed4138d3a8e3599/src/completions/Tab.ts#L155

You need to add the config setting in src/lib/config.ts (e.g. tabsort mru|default) and import the config into the tab completions. Then use config.get("tabsort"). 

Please ask on Matrix if you have any questions :)

**Comment by berkeleyinc** [comment]
Thank you! Your description what to change was on point!
Pull request is here: https://github.com/tridactyl/tridactyl/pull/2133

**Comment by bovine3dom** [comment]
Added in #2133. Thanks @berkeleyinc!

Simply `:set tabsort mru` in the latest betas.

**Comment by x-ji** [comment]
Thanks for the PR. Just installed `tridactyl` again and giving it another go now :)
